src/App.js

```diff
@@ -6,20 +6,24 @@ import Register from './components/Register';
 import Login from './components/Login/Login';
 import GuestHomepage from './components/guest-homepage/GuestHomepage';
 import ProfilePage from './components/my-profile-page/ProfilePage';
+
+import Layout from './components/Layout/Layout';
+
 import HomepageAuthUser from './components/homepage-authuser/HomepageAuthUser';
 
+
 function App() {
     return (
         <div className="App">
             <BrowserRouter>
-                <Navbar />
-                <Routes>
-                    <Route path="/login" element={<Login />} />
-                    <Route path="/register" element={<Register />} />
-                    <Route path="/profile" element={<ProfilePage />} />
-                    <Route path="/board" element={<GuestHomepage />} />
-                    <Route path="/team" element={<HomepageAuthUser />} />
-                </Routes>
+                <Layout>
+                    <Routes>
+                        <Route path="/login" element={<Login />} />
+                        <Route path="/register" element={<Register />} />
+                        <Route path="/profile" element={<ProfilePage />} />
+                        <Route path="/board" element={<GuestHomepage />} />
+                    </Routes>
+                </Layout>
```

Comment:
I think we should do this a bit differently, we want to be able to specify the layout per route, or per route group - for example, for Login, Register and GuestHomepage components, we want to have a different layout component, e.g. GuestLayout, and for ProfilePage and other auth pages, we can have AuthLayout

Let's create a generic `<Page>` component that will have `layout` and `content` as properties, so we can have something like:

```jsx
<Route path="/board" element={<Page layout={GuestLayout} content={GuestHomepage} />
```

Reply:
I have tried with <Page> but couldn't manage to handle it arround. In new push I divided routes on GuestLayout & AuthLayout to group targeted routes.

Reply:
I think it's done now :)

Reply:
I think it is done now :)